src/jvm/main/org/jetbrains/kotlinx/lincheck/Utils.kt

```diff
@@ -49,10 +49,9 @@ internal fun executeActor(
         val res = m.invoke(instance, *args.toTypedArray())
         return if (m.returnType.isAssignableFrom(Void.TYPE)) VoidResult else createLincheckResult(res)
     } catch (invE: Throwable) {
-        val eClass = (invE.cause ?: invE).javaClass.normalize()
-        for (ec in actor.handledExceptions) {
-            if (ec.isAssignableFrom(eClass))
-                return ExceptionResult.create(eClass)
+        val exception = invE.cause ?: invE
+        if (exceptionCanBeValidExecutionResult(exception)) {
```

Comment:
Is this statement always "true"?

Reply:
Under some circumstances some exception can be thrown here, and I think it's good that we will process it in the same way

Reply:
I agree, but which exceptions are invalid here?